experiments/run-big-bench.py

```diff
@@ -136,9 +136,18 @@ def compute_metrics(
         QPS over all queries
 
     """
-    start = time.time()
-    _, top_k_indices = index.search(queries=queries, ef_search=ef_search, K=k)
-    end = time.time()
+    if type(index) in (flatnav.index.L2Index, flatnav.index.IPIndex):
```

Comment:
Is there a simpler check we can do here. Like check if the index is hnsw instead of flatnav?

Reply:
The problem with checking the index name is that we have to pass the index name argument to the function just so we know which index type it is, but we can get that information from the object itself.